duckduckgo/iOS
# Update NetP settings item to match design

Task/Issue URL: https://app.asana.com/0/0/1205084446087077/f
Tech Design URL:
CC:

**Description**:

For [iOS: Update NetworkProtection package to support iOS](https://app.asana.com/0/0/1204188117085830) we added a settings menu item to access NetP but it was done to a speculative design and with the incorrect fonts.

This task should:
Update the fonts to match the other settings items
Show Connected or Not connected depending on the PacketTunnelProvider`s status.

For the designs, check https://app.asana.com/0/0/1205084446087077/f



**Steps to test this PR**:
1. Check out this branch
2. Go to Build Phases and in Embed App Extensions, add the PacketTunnelProvider
3. Run on a real device
4. Ensure you’re authenticated as internal
5. Navigate to the Settings
6. **Check the Network Protection settings item is visible, says Not Connected and matches designs**
7. Tap the item to navigate to the NetP status view
8. If you have no token (there should be a message saying “Already Redeemed”), enter an[ invite code](https://app.asana.com/0/1203135672790568/1204303483337129/f)
9. Flip the switch to activate Network Protection
10. Navigate back to the settings screen
11. **Check the Network Protection settings item says Connected and matches designs**

<!—
Before submitting a PR, please ensure you have tested the combinations you expect the reviewer to test, then delete configurations you *know* do not need explicit testing.

Using a simulator where a physical device is unavailable is acceptable.
—>

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
